config/airkeeper.example.json

```diff
@@ -5,30 +5,56 @@
     {
       "id": "31337",
       "contracts": {
-        "RrpBeaconServer": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788"
+        "RrpBeaconServer": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
+        "AirnodeProtocol": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+        "DapiServer": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
       }
     }
   ],
   "triggers": {
     "rrpBeaconServerKeeperJobs": [
       {
         "chainIds": ["31337"],
-        "templateId": "0x50c604914d8ed35473149457a1a0912b785813b4e2e51bd2b75409ca25c50e1d",
-        "templateParameters": [
-          { "type": "bytes32", "name": "to", "value": "USD" },
-          { "type": "bytes32", "name": "_type", "value": "int256" },
-          { "type": "bytes32", "name": "_path", "value": "result" },
-          { "type": "bytes32", "name": "_times", "value": "1000000" }
-        ],
-        "overrideParameters": [
-          { "type": "bytes32", "name": "from", "value": "ETH" }
-        ],
+        "templateId": "0x6f737bbf31dfed584a16f53b7d725ff64bee67e79a468259456fb40aa19c60c4",
+        "overrideParameters": [{ "type": "string32", "name": "from", "value": "ETH" }],
         "oisTitle": "Currency Converter API",
         "endpointName": "convertToUSD",
         "deviationPercentage": "5",
         "keeperSponsor": "0x2479808b1216E998309A727df8A0A98A1130A162",
         "requestSponsor": "0x61648B2Ec3e6b3492E90184Ef281C2ba28a675ec"
       }
+    ],
+    "proto-psp": [
+      {
+        "subscriptionId": "0xdf6d2785e1fc71ba3aa08fccb77d74ac4ba678208bc33c445152ea999a0cc8af",
+        "overrideParameters": [{ "type": "string32", "name": "from", "value": "ETH" }],
+        "oisTitle": "Currency Converter API",
+        "endpointName": "convertToUSD"
```

Comment:
Right I was struggling with placing these 2 values and ended up replicating oisTitle and endpointName just to be consistent with airnode config.json rrp triggers.

I'll add the endpoints object in the root of the file but since this is a breaking change for old rrp beacon update function, should we consider renaming `rrpBeaconServerKeeperJobs` or some other property in the airkeeper.json config?

Reply:
I don't think that we have to worry about breaking Airkeeper